Makefile

```diff
@@ -238,6 +238,7 @@ protogen_clean: ## Remove all the generated protobufs.
 
 .PHONY: protogen_local
 protogen_local: go_protoc-go-inject-tag ## Generate go structures for all of the protobufs
+# TODO: Organize this code with a basic fo
```

Comment:
```suggestion
# TODO: Organize this code with a basic for loop
```
?

P.S. I believe that @Gustavobelfort was looking into integrating `buf` and at that point protobuf generation would be more declarative and without all these calls. Just flagging